common/model-views.cpp

```diff
@@ -560,7 +571,13 @@ namespace rs2
             if (opt == RS2_OPTION_HOLES_FILL)
                 use_option_name = false;
 
-            auto desc = endpoint->get_option_description(opt);
+            bool should_be_displayed_in_cm = false;
+            std::string device_pid = dev->s->get_info(RS2_CAMERA_INFO_PRODUCT_ID);
+            if (device_pid == "0B5B" && (opt == RS2_OPTION_MIN_DISTANCE || opt == RS2_OPTION_MAX_DISTANCE || opt == RS2_OPTION_DEPTH_UNITS))
+                should_be_displayed_in_cm = true;
+
+            auto desc_str = get_option_description(opt, should_be_displayed_in_cm);          
+            auto desc = desc_str.c_str();
```

Comment:
Return the original call and invoke the 2nd decorator call to adjust units

Reply:
Done